src/blog/users/service.py

```diff
@@ -1,7 +1,7 @@
-from blog.db.db import user_table
-from blog.users.models import User, UserCreate, UserUpdate, UserRead
+from src.blog.db.db import user_table
+from src.blog.users.models import User, UserCreate, UserUpdate, UserRead
 import typing as t
-from blog.utils import max_id
+from src.blog.utils import max_id
```

Comment:
As we have a database now, max_id is not needed anymore. Database should be configured to auto increment new values. 
That's why max_id function is not needed.

Reply:
Deleted. 👍🏻